Formula/yarn.rb

```diff
@@ -1,26 +1,19 @@
-require "language/node"
-
 class Yarn < Formula
   desc "JavaScript package manager"
   homepage "https://yarnpkg.com/"
   url "https://yarnpkg.com/downloads/0.21.3/yarn-v0.21.3.tar.gz"
   sha256 "0946a4d1abc106c131b700cc31e5c3aa5f2205eb3bb9d17411f8115354a97d5d"
```

Comment:
You'll need to move `revision` above `head` or it won't pass `brew audit --strict`